packages/medusa/src/api/routes/admin/order-edits/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Router } from "express"
+import middlewares from "../../../middlewares"
+import { isFeatureFlagEnabled } from "../../../middlewares/feature-flag-enabled"
+import OrderEditingFeatureFlag from "../../../../loaders/feature-flags/order-editing"
+import { DeleteResponse } from "../../../../types/common"
+import { FlagRouter } from "../../../../utils/flag-router"
+
+const route = Router()
+
+export default (app, featureFlagRouter: FlagRouter) => {
```

Comment:
todo: there is no need to pass it as an arguments since we can directly import it. wdyt? also it is not used.